packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.tsx

```diff
@@ -267,10 +266,18 @@ const FinancialAssessment = observer(() => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    const onSubmit = async (values: FormikValues, { setSubmitting, setStatus }: FormikHelpers<any>) => {
+    const onSubmit = async (
+        values: TFinancialInformationForm,
+        { setSubmitting, setStatus }: FormikHelpers<TFinancialInformationForm>
+    ) => {
         setStatus({ msg: '' });
         setIsBtnLoading(true);
-        const form_payload: any = {
+
+        if (shouldHideOccupationField(values.employment_status || employment_status)) {
+            delete values?.occupation;
+        }
+
+        const form_payload: { financial_information: TFinancialInformationForm } = {
             financial_information: { ...values },
         };
         const data = await setFinancialAndTradingAssessment(form_payload);
```

Comment:
There's a type mismatch error between `form_payload` which I believe is due to the type `GetFinancialAssessment` being used in state above  
Can we have a look at the type or have discussion with BE team to sync these else we need to replace the type in state above